apis/fluent-en_GB/extensions/jdk8/atrium-api-fluent-en_GB-jdk8-jvm/src/main/kotlin/ch/tutteli/atrium/api/fluent/en_GB/jdk8/pathAssertions.kt

```diff
@@ -33,6 +33,18 @@ fun <T : Path> Expect<T>.exists(): Expect<T> = addAssertion(ExpectImpl.path.exis
  */
 fun <T : Path> Expect<T>.existsNot(): Expect<T> = addAssertion(ExpectImpl.path.existsNot(this))
 
+
+/**
+ * Expects that the subject of the assertion (a [Path]) is ends with the other [Path];
+ *
+ * @param other The [Path] provided to the assertion
+ * @return This assertion container to support a fluent API.
+ * @throws AssertionError Might throw an [AssertionError] if the assertion made is not correct.
+ *
+ * @since 0.9.0
+ */
+fun <T : Path> Expect<T>.endsWith(other: T): Expect<T> = addAssertion(ExpectImpl.path.endsWith(this, other))
```

Comment:
We always use `expected`, let's stick to this wording
```suggestion
fun <T : Path> Expect<T>.endsWith(expected: T): Expect<T> = addAssertion(ExpectImpl.path.endsWith(this, expected))
```